packaging/standalone/src/main/distribution/shell-scripts/bin/neo4j

```diff
@@ -215,9 +215,20 @@ setup_java_opts() {
   [[ "${wrapper_java_initmemory:-}" ]] && JAVA_MEMORY_OPTS="${JAVA_MEMORY_OPTS} -Xms${wrapper_java_initmemory}m"
   [[ "${wrapper_java_maxmemory:-}" ]] && JAVA_MEMORY_OPTS="${JAVA_MEMORY_OPTS} -Xmx${wrapper_java_maxmemory}m"
 
+  JAVA_GC_LOGGING_OPTS=""
+  if [ "${dbms_logs_gc_enabled:-}" = "true" ]; then
+      JAVA_GC_LOGGING_OPTS="-Xloggc:data/log/gc.log ${dbms_logs_gc_options}";
+  fi
+  if [ "${dbms_logs_rotation_size:-}" -a "${dbms_logs_rotation_keep_number:-}" ]; then
+      JAVA_GC_LOGGING_OPTS="${JAVA_GC_LOGGING_OPTS} -XX:+UseGCLogFileRotation -XX:NumberOfGCLogFiles=${dbms_logs_rotation_keep_number} -XX:GCLogFileSize=${dbms_logs_rotation_size}";
+  fi
```

Comment:
Annoyingly you have to duplicate all the defaults here (for the options and rotation policy). We don't have a way to read config in the scripts which respects the defaults.